HW5/HW5.py

```diff
@@ -0,0 +1,627 @@
+                <p>home_def_single_avg, away_off_double_avg, home_off_batting_avg,
+                away_off_walk_to_strikeout_avg, away_off_single_avg, away_def_double_avg</p>
+
+                <h2>Correlations</h2>
+                <h3>Correlation Table (Sorted by PCC DESC)</h3>
+                {feature_correlation_table_html}
+                <p>PCC is Pearson correlation coefficient. The higher PCC,
+                the more significant relationship between predictors.</p>
+                <p>PCC is closer to 0 which means that there are less reationship between predictors.</p>
+                <p>Based on this, home_def_batting_avg,
+                away_def_ground_to_flyout_avg, home_def_ground_to_flyout_avg,</p>
+                <p>home_off_batting_avg, away_off_ground_to_flyout_avg,
+                away_def_plateapperance_strikeout, away_off_ground_to_flyout_avg</p>
+                <p>away_def_batting_avg, home_off_walk_to_strikeout_avg,
+                home_off_ground_to_flyout_avg</p>
+
+                <a href='{feature_correlation_matrix_html}'><img src='{feature_correlation_matrix_image}'/></a>
+                <p>The coefficient is same with the correlation table</p>
+
+                <h2>Brute-Force Mean Diff (Sorted by Weighted DESC)</h2>
+                {feature_2d_diff_mean_table_html}
+                <p>We need to use one table to explore the different with mean
+                between one predictor with response and to select the features which influence
+                response more than others</p>
+
+                <h2>Feature analysis</h2>
+                <p>According to the results I got from each table, I decide to use the following
+                feature to fit the model and see the different score</p>
+                <p>home_def_ground_to_flyout_avg, away_def_ground_to_flyout_avg,
+                home_off_ground_to_flyout_avg, away_off_ground_to_flyout_avg, home_def_walk_to_strikeout_avg,</p>
+                <p>home_off_ground_to_flyout_avg, home_off_plateapperance_strikeout,
+                away_off_plateapperance_strikeout, home_off_walk_to_strikeout_avg,</p>
+                <p>home_def_plateapperance_strikeout, home_off_batting_avg, away_off_batting_avg, away_def_batting_avg,
+                home_def_batting_avg</p>
+                <p>After running the selected features, the average score is lower than the
+                score for all features so I decide to use the all features I have.</p>
+
+                <h2>Selected Features:</h2>
+                {selected_features_html}
+
+                <h2>Model Scoring Comparison</h2>
+                {model_scoring_table_html}
+
+                <h2>Conclusion</h2>
+                <p>Based on Model Scoring Comparison Table, Logistic regression has better
+                performace score than other models</p>
+
+            </body>
+        </html>
+    """
+
+    f.write(html)
+    f.close()
+
+
+if __name__ == "__main__":
+    sys.exit(main())
```

Comment:
which model do you think is better?

Reply:
I have a comment in my HTML, and I think logistical regression is better than others